translations/fr/try_ruby_10.md

```diff
@@ -0,0 +1,26 @@
+---
+lang:   EN
+title:  Vous avez 30 minutes? Essayez Ruby!
+answer:
+class:  stretcher chapmark
+ok:
+error:
+---
+
+Ruby est un language de programmation originaire du Japon et qui a
+revolutionné le développement logiciel.
+
+La beauté de Ruby se situe dans sa balance entre simplicité et puissance.
+
+Vous pouvez saisir du code Ruby dans l'éditeur et utiliser les boutons suivants pour naviguer:
+
+- __Run__ &rarr; Execute le code dans l'éditeur
+- __Copy__ &rarr; Copie l'example de code dans l'éditeur
+- __Next__ &rarr; Vous permet de sauter à la leçon suivante
+- __Back__ &rarr; Vous permet de revenir à la leçon précédente
+- __Clear__ &rarr; Remets l'éditeur à zéro
+
+Utilise le bouton 'planête' <span aria-hidden="true" class="glyphicon glyphicon-globe"></span>
+ci-dessus pour changer le language de TryRuby.
+
+### Clique sur __Next__ pour commencer à apprendre.
```

Comment:
If we are to use the `tu` (the friendly choice), this should be used everywhere instead of the `vous` (the formal choice).

Reply:
Absolutely, I'd rather default to friendlier. Québécois-style 😸